<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kaava</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="general_info_tab">
      <attribute name="title">
       <string>Perustiedot</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QScrollArea" name="general_info_scroll_area">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>831</width>
            <height>562</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="leftMargin">
            <number>9</number>
           </property>
           <property name="rightMargin">
            <number>9</number>
           </property>
           <property name="bottomMargin">
            <number>9</number>
           </property>
           <item>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Pysyvä kaavatunnus:</string>
               </property>
               <property name="buddy">
                <cstring>permanent_identifier_line_edit</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="permanent_identifier_line_edit">
               <property name="toolTip">
                <string>Kaava-asian pysyvä kaavatunnus</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="nimiLabel">
               <property name="text">
                <string>Nimi*:</string>
               </property>
               <property name="buddy">
                <cstring>name_line_edit</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="name_line_edit">
               <property name="toolTip">
                <string>Kaava-asian nimi</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="kuvausLabel">
               <property name="text">
                <string>Kuvaus:</string>
               </property>
               <property name="buddy">
                <cstring>description_text_edit</cstring>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QTextEdit" name="description_text_edit">
               <property name="toolTip">
                <string>Kaava-asian kuvaus</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="tyyppiLabel">
               <property name="text">
                <string>Kaavalaji*:</string>
               </property>
               <property name="buddy">
                <cstring>plan_type_combo_box</cstring>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="HierarchicalCodeComboBox" name="plan_type_combo_box">
               <property name="toolTip">
                <string>Kaava-asian kaavalaji</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="elinkaarenTilaLabel">
               <property name="text">
                <string>Elinkaaren tila*:</string>
               </property>
               <property name="buddy">
                <cstring>lifecycle_status_combo_box</cstring>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="CodeComboBox" name="lifecycle_status_combo_box">
               <property name="toolTip">
                <string>Kaavan elinkaaren tila</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Organisaatio*:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="CodeComboBox" name="organisation_combo_box"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QgsCollapsibleGroupBox" name="mGroupBox">
             <property name="title">
              <string>Muut tunnukset</string>
             </property>
             <property name="collapsed">
              <bool>true</bool>
             </property>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Diaarinumero:</string>
                </property>
                <property name="buddy">
                 <cstring>record_number_line_edit</cstring>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="record_number_line_edit"/>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="producers_plan_identifier_line_edit"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Tuottajan kaavatunnus:</string>
                </property>
                <property name="buddy">
                 <cstring>producers_plan_identifier_line_edit</cstring>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Asianhallintatunnus:</string>
                </property>
                <property name="buddy">
                 <cstring>matter_management_identifier_line_edit</cstring>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="matter_management_identifier_line_edit"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="general_regulations_tab">
      <attribute name="title">
       <string>Yleismääräykset</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Yleismääräykset</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Kaavamääräysryhmäkirjastot</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="plan_regulation_group_libraries_combobox"/>
            </item>
            <item>
             <widget class="QgsFilterLineEdit" name="mLineEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="placeholderText">
               <string>Suodata kaavamääräysryhmiä</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
              <property name="showSearchIcon">
               <bool>true</bool>
              </property>
              <property name="qgisRelation" stdset="0">
               <string notr="true"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTreeWidget" name="plan_regulation_groups_tree">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="dragDropMode">
               <enum>QAbstractItemView::NoDragDrop</enum>
              </property>
              <property name="alternatingRowColors">
               <bool>false</bool>
              </property>
              <attribute name="headerVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string/>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Valitut yleismääräysryhmät</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QScrollArea" name="plan_regulation_group_scrollarea">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="plan_regulation_group_scrollarea_contents">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>541</width>
                 <height>493</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6"/>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsFilterLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgsfilterlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>CodeComboBox</class>
   <extends>QComboBox</extends>
   <header>arho_feature_template.gui.components.code_combobox</header>
  </customwidget>
  <customwidget>
   <class>HierarchicalCodeComboBox</class>
   <extends>QComboBox</extends>
   <header>arho_feature_template.gui.components.code_combobox</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>general_info_scroll_area</tabstop>
  <tabstop>name_line_edit</tabstop>
  <tabstop>description_text_edit</tabstop>
  <tabstop>plan_type_combo_box</tabstop>
  <tabstop>lifecycle_status_combo_box</tabstop>
  <tabstop>mGroupBox</tabstop>
  <tabstop>record_number_line_edit</tabstop>
  <tabstop>producers_plan_identifier_line_edit</tabstop>
  <tabstop>matter_management_identifier_line_edit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
